src/sentry/snuba/metrics/query_builder.py

```diff
@@ -179,9 +175,12 @@ def __init__(self, query_params, paginator_kwargs: Optional[Dict] = None):
             raise InvalidField(
                 "We don't have more than 250 buckets stored for any given metric bucket."
             )
-        self.histogram_from = query_params.get("histogramFrom", None)
-        self.histogram_to = query_params.get("histogramTo", None)
+        histogram_from = query_params.get("histogramFrom", None)
+        histogram_to = query_params.get("histogramTo", None)
+        self.histogram_from = int(histogram_from) if histogram_from is not None else None
+        self.histogram_to = int(histogram_to) if histogram_to is not None else None
```

Comment:
`rebucket_histograms` take floats, so I guess these should be floats, too? Or is there a special reason to only accept ints?

```suggestion
        self.histogram_from = float(histogram_from) if histogram_from is not None else None
        self.histogram_to = float(histogram_to) if histogram_to is not None else None
```

Are we fine with responding with status code 500 when I pass `histogramFrom=foo`?

Reply:
>Are we fine with responding with status code 500 when I pass histogramFrom=foo?

I think we should fix those cases once ppl complain.. there are too many